From: NIIBE Yutaka <gniibe@fsij.org>
Date: Mon, 15 Jun 2020 11:32:08 +0900
Subject: build: Fix the gpg-error-config test for cross build.

* src/gpg-error-config-test.sh.in: Clear PKG_CONFIG_SYSROOT_DIR.

GnuPG-bug-id: 4973
Signed-off-by: NIIBE Yutaka <gniibe@fsij.org>
(cherry picked from commit 1a4c1f37a1c95a4cfcaab6df3364ee3515a1ea1b)
---
 src/gpg-error-config-test.sh.in | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/gpg-error-config-test.sh.in b/src/gpg-error-config-test.sh.in
index 02bc881..b888aa2 100644
--- a/src/gpg-error-config-test.sh.in
+++ b/src/gpg-error-config-test.sh.in
@@ -8,6 +8,8 @@ export PKG_CONFIG_PATH
 
 if [ "$1" = --old-new ]; then
     PKG_CONFIG_CMD=./gpg-error-config-old
+    # Old gpg-error-config never supports PKG_CONFIG_SYSROOT_DIR
+    unset PKG_CONFIG_SYSROOT_DIR
 else
     pkg_config_min_version=0.29
     PKG_CONFIG_SYSROOT_DIR="/var/example-target"
